spec/requests/delete_item_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+describe 'delete list item  api' do
+  it 'deletes a list item' do
+
+    user = create(:user)
+    api_key = user.api_key
+    list1 = create(:list, :title => 'List 1', :user => user)
+    item = create(:item, :name => 'Item', :list => list1)
+
+    delete "/api/v1/lists/#{list1.id}/items/#{item.id}.json", nil, "X-Api-Key" => api_key
```

Comment:
The `.json` on the URL path shouldn't be needed in this spec and the lower down request specs.